eng/pipelines/common/platform-matrix.yml

```diff
@@ -219,7 +219,7 @@ jobs:
       targetRid: browser-wasm
       platform: Browser_wasm
       container:
-        image: ubuntu-18.04-webassembly-20210223133559-4800846
+        image: ubuntu-18.04-webassembly-20210309143118-005aab4
```

Comment:
@steveisok @mdh1418 Do you know why this change was made? `main` is still using the older version.

Reply:
judging from the date it's to bring in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/commit/f25f6cd55e437581a96448d8f3b66c377b95fa8a